protobuf/lib/src/protobuf/builder_info.dart

```diff
@@ -113,13 +113,24 @@ class BuilderInfo {
   }
 
   // Repeated message, group, or enum.
+  void pc<T>(int tagNumber, String name, int fieldType,
+      [CreateBuilderFunc subBuilder,
+      ValueOfFunc valueOf,
+      List<ProtobufEnum> enumValues]) {
+    assert(_isGroupOrMessage(fieldType) || _isEnum(fieldType));
+    addRepeated<T>(tagNumber, name, fieldType, _checkNotNull, subBuilder,
+        valueOf, enumValues);
+  }
+
+  // Deprecated, use [pc] instead.
+  @deprecated
```

Comment:
I think there is a deprecated constructor that takes the description as an argument https://api.dartlang.org/stable/2.1.0/dart-core/Deprecated-class.html

Reply:
You can also mention in the docstring that the check is ignored.